util/execdetails/execdetails.go

```diff
@@ -253,11 +253,47 @@ func (crs *CopRuntimeStats) String() string {
 		procTimes[n-1], procTimes[0], procTimes[n*4/5], procTimes[n*19/20], totalRows, totalIters, totalTasks)
 }
 
+// ReaderRuntimeStats collects stats for TableReader, IndexReader and IndexLookupReader
+type ReaderRuntimeStats struct {
+	sync.Mutex
+
+	copRespTime []time.Duration
+}
+
+// RecordOneCopTask record once cop response time to update maxcopRespTime
+func (rrs *ReaderRuntimeStats) RecordOneCopTask(t time.Duration) {
+	rrs.Lock()
+	defer rrs.Unlock()
+	rrs.copRespTime = append(rrs.copRespTime, t)
+}
+
+func (rrs *ReaderRuntimeStats) String() string {
+	sort.Slice(rrs.copRespTime, func(i, j int) bool {
+		return rrs.copRespTime[i] < rrs.copRespTime[j]
+	})
+	s := "cop resp time "
```

Comment:
Can we make it shorter? How about the same form as `proc time` and call it `rpc time`?

Reply:
That's OK.